content/zh/docs/contribute/localization_zh.md

```diff
@@ -0,0 +1,463 @@
+### Create a Pod
+-->
+### 创建 Pod   {#create-a-pod}
+```
+
+此类问题对于概念部分的页面最为突出，需要格外注意。
+
+
+### 中文链接目标
+
+由于大部分页面已经完成中文本地化，这意味着很多链接可以使用中文版本作为目标。
+例如：
+
+```
+<!--
+For more information, please check [volumes](/docs/concepts/storage/)
+...
+-->
+更多的信息可参考[卷](/zh/docs/concepts/storage/)页面。
+```
+
+如果对应目标页面尚未本地化，建议登记一个 Issue。
+
+{{< note >}}
+Website 的仓库中 `scripts/linkchecker.py` 是一个工具，可用来检查页面中的链接。
+例如，下面的命令检查中文本地化目录 `/content/zh/docs/concepts/containers/`
+中所有 Markdown 文件中的链接合法性：
+
+```
+./scripts/linkchecker.py -l zh -f /docs/concepts/containers/**/*.md
+```
+{{< /note >}}
+
+## 排版格式
+
+以下为译文 Markdown 排版格式要求：
+
+- 中英文之间留一个空格
+  * 这里的“英文”包括以英文呈现的超级链接
+  * 这里的中文、英文都不包括标点符号
+- 译文 Markdown 中不要使用长行，应适当断行。
+  * 可根据需要在 80-120 列断行
+  * 最好结合句子的边界断行，即一句话在一行，不必留几个字转到下一行
+  * 超级链接文字一般较长，可独立成行
+
+{{< warning >}}
+我们注意到有些贡献者可能使用了某种自动化工具，在 Markdown 英文原文中自动
+添加空格。虽然这些工具可一定程度提高效率，仍然需要提请作者注意，某些工具
+所作的转换可能是不对的，例如将 `foo=bar` 转换为 `foo = bar`、将
+`），另一些文字` 转换为 `） ，另一些文字` 等等，甚至将超级链接中的半角
+井号（`#`）转换为全角，导致链接失效。
+{{< /warning >}}
+
+英文中 "you" 翻译成 "你" 不必是 “您"
+文章内的链接用英文例如 (#deploying )，在对应的标题上后面加上 {#deploying}
+
+
+## 术语
+
+### 术语拼写
+
+按中文译文习惯，尽量不要在中文译文中使用首字母小写的拼写。例如：
+
+```
+列举所有 pods，查看其创建时间 ...       [No]
+列举所有 Pod，查看其创建时间 ...        [Yes]
+```
+
+*第一次*使用首字母缩写时，应标注其全称和中文译文。例如：
+
+```
+你可以创建一个 Pod 干扰预算（Pod Disruption Budge，PDB）来解决这一问题。
+所谓 PDB 实际上是 ...
+```
+
+对于某些特定于 Kubernetes 语境的术语，也应在*第一次*出现在页面中时给出其英文原文，
+以便读者对照阅读。例如：
+
+```
+镜像策略（Image Policy）用来控制集群可拉取的镜像仓库（Image Registry）源。
+```
+
+### 术语对照
+
+本节列举常见术语的统一译法。除极个别情况，对于专业术语应使用本节所列举的译法：
+
+- Addons，插件
+- API Server，API 服务器
+- Bearer Token，持有者令牌
+- GA (general availability)，正式发布
+- Headless Service，Headless Service
+- admission controller，准入控制器
+- affinity，亲和性
+- annotation，注解
+- anti-affinity，反亲和性
+- attach，挂接
+- autoscale，自动扩缩容
+- capabilities，权能字
+- certificate authority，证书机构
+- certificate，证书
+- claim，申领
+- cloud provider
+  * 当用来指代下层云服务的提供厂商时，译为云服务提供商
+  * 当特指 Kubernetes 中对不同云平台的支持时，译为云驱动
+- cluster，集群
+- condition
+  * 大多数上下文中，可译为“条件”
+  * 在讨论 Kubernetes 资源的 condition 时，应译为“状况”
+- control loop，控制回路
+- control plane，控制平面，或控制面
+- controller manager，控制器管理器
+- credential，登录凭据，凭据
+- custom，定制，或自定义
+- daemon，守护进程
+- dashboard，仪表板
+- dependent，附属或附属者
+- deprecated，已弃用的
+- deprecation，弃用
+- desired，预期的
+- desired state，预期状态
+- detach，解除挂接
+- distribution，发行版本
+- disruption，干扰（请勿译为“中断”）
+- drain，腾空
+- endpoint，端点
+- egress，出站
+- evict，驱逐
+- eviction，驱逐
+- feature gate，特性门控
+- federation，联邦
+- flags，命令行参数，参数
+- grace period，宽限期限
+- graceful termination，体面终止
+- hairpin，发夹
+- hash，哈希
+- headless service，无头服务
+- healthcheck，健康检查
+- hook，回调
+- host，主机，宿主机
+- hosting，托管
+- idempotent，幂等的
+- image，镜像
+- image registry，镜像仓库
+- ingress，入站
+- init container，Init 容器
+- key
+  * 在加密解密、安全认证上下文中，译为密钥
+  * 在配置文件、数据结构上下文中，译为主键，或键
+- label，标签
+- label selector，标签选择算符
+- lifecycle，生命周期
+- limit，限制，限值
+- liveness probe，存活态探测
+- load balance，负载均衡
+- load balancer，负载均衡器
+- log flush，清刷日志数据
+- loopback，本地回路
+- manifest，清单，清单文件
+- master node，主控节点
```

Comment:
I would not include this line. Kubernetes does not have master nodes. It _can_ have nodes where control place components are running, and there is a deprecated / problematic label `node-role.kubernetes.io/master` used by some tools such as `kubeadm`.